### PR TITLE
return fasta sequence in response if passed

### PR DESCRIPTION
**Pull request by ahmadshadab**

### CODE REVIEW

File: core-parser/src/test/java/org/uniprot/core/parser/fasta/uniprot/UniProtKBFastaParserTest.java [hunk 1]
```diff
@@ -238,6 +237,27 @@ void testToUniProtKBFasta(){
...
+        assertEquals(entry.getUniProtkbId(), result.getUniProtkbId());
+        assertEquals("Rec Name Value", result.getProteinName());
+        assertEquals("ORF 1", result.getGeneName());
+        assertEquals(entry.getOrganism(), result.getOrganism());
+        assertEquals("FRAGMENTS_PRECURSOR", result.getFlagType().name());
+        assertEquals(entry.getProteinExistence(), result.getProteinExistence());
+        assertEquals(entry.getEntryAudit().getSequenceVersion(), result.getSequenceVersion());
+        assertEquals(entry.getSequence().getValue().substring(9, 20), result.getSequence().getValue());
+        assertEquals(sequenceRange, result.getSequenceRange());
     }
```
Comment (LeonardoGonzales) [inline core-parser/src/test/java/org/uniprot/core/parser/fasta/uniprot/UniProtKBFastaParserTest.java:261]: Can you please add test for subsequence out of range? 
For example, in a sequence with length 50, the user requests a subsequence [60-90] both out of range or [40-80] just end out of range?

File: core-domain/src/main/java/org/uniprot/core/genecentric/impl/ProteinImpl.java [hunk 2]
```diff
@@ -18,7 +18,7 @@ public class ProteinImpl extends UniProtKBFastaImpl implements Protein {
     private static final long serialVersionUID = -2231166801909993211L;
```
[the remark this answers is not in the document]
Comment (ahmadshadab) [reply core-domain/src/main/java/org/uniprot/core/genecentric/impl/ProteinImpl.java:18]: Added test in ProteinBuilderTest. We have one test in class ProteinImplTest which goes through no-arg constructor and test object creation.

**Review by LeonardoGonzales — APPROVED** [review]
